draft-ietf-core-problem-details.md

```diff
@@ -235,9 +236,10 @@ The same example but using a registered unsigned int as
 
 ~~~
 {
-  / title /       -1: "title of the error",
-  / detail /      -2: "detailed information about the error",
-  / instance /    -3: "coaps://pd.example/FA317434",
+  / title /         -1: "title of the error",
+  / detail /        -2: "detailed information about the error",
+  / instance /      -3: "coaps://pd.example/FA317434",
+  / response-code / -4: 128, / 4.00 /
```

Comment:
Although correct, that can be quite confusing.
(We have h32 in ED for byte strings, but no base32 for numbers.)
Hmm.

Reply:
:juggling_person: